bundles/org.openhab.binding.shelly/README.md

```diff
@@ -1397,6 +1398,15 @@ Channels lastEvent and eventCount are only available if input type is set to mom
 |       | totalKWH     | Number   | yes       | Total energy consumption in kwh since the device powered up (resets on restart)   |
 |       | lastUpdate   | DateTime | yes       | Timestamp of the last measurement                                                 |
 
+### Shelly Pro 1CB (thing-type: shellypro1cb)
+
+| Group | Channel      | Type     | read-only | Description                                                                      |
+| ----- | ------------ | -------- | --------- | -------------------------------------------------------------------------------- |
+| relay | output       | Switch   | r/w       | Controls the relay's output channel (on/off)                                     |
+|       | outputName   | String   | yes       | Logical name of this relay output as configured in the Shelly App                |
+| meter | voltage       | Number   | yes       | RMS voltage, Volts                                                              |
```

Comment:
The Thing seems to be created with channel groups `relay`, `sensors` and `device`. So it probably doesn't contain channel group `meter`?

Reply:
It will be created with groups device, relay and meter
```
<thing-type id="shellyprodimmer1pm">
		<label>Shelly Pro Dimmer 1PM</label>
		<description>@text/thing-type.shelly.shellyprodimmer1pm.description</description>
		<semantic-equipment-tag>Lightbulb</semantic-equipment-tag>
		<channel-groups>
			<channel-group id="relay" typeId="dimmerChannel" />
			<channel-group id="meter" typeId="meter" />
			<channel-group id="device" typeId="deviceStatus" />
		</channel-groups>

		<representation-property>serviceName</representation-property>
		<config-description-ref uri="thing-type:shelly:rgbw2" />
	</thing-type>
```

Reply:
It will be created with groups device, relay, meter

```
<thing-type id="shellyprodimmer1pm">
		<label>Shelly Pro Dimmer 1PM</label>
		<description>@text/thing-type.shelly.shellyprodimmer1pm.description</description>
		<semantic-equipment-tag>Lightbulb</semantic-equipment-tag>
		<channel-groups>
			<channel-group id="relay" typeId="dimmerChannel" />
			<channel-group id="meter" typeId="meter" />
			<channel-group id="device" typeId="deviceStatus" />
		</channel-groups>

		<representation-property>serviceName</representation-property>
		<config-description-ref uri="thing-type:shelly:rgbw2" />
	</thing-type>
```

Reply:
This PR introduces thing type `shellypro1cb`:
```xml
	<thing-type id="shellypro1cb">
		<label>ShellyPro 1CB</label>
		<description>@text/thing-type.shelly.shellypro1cb.description</description>
		<semantic-equipment-tag>ControlDevice</semantic-equipment-tag>
		<channel-groups>
			<channel-group id="relay" typeId="relayChannel"/>
			<channel-group id="sensors" typeId="externalSensors"/>
			<channel-group id="device" typeId="deviceStatus"/>
		</channel-groups>

		<representation-property>serviceName</representation-property>
		<config-description-ref uri="thing-type:shelly:relay-gen2"/>
	</thing-type>
```

?